src/Athens-Examples/AthensTextRenderManualTests.class.st

```diff
@@ -64,7 +64,8 @@ AthensTextRenderManualTests class >> test2 [
 		canvas pathTransform rotateByDegrees: 0.
 		c renderOn: canvas
 	].
-	Display getCanvas translucentImage: surf asForm at: 0@0
+
+	surf asForm asMorph openInWindow
```

Comment:
Could it be

`surf asMorph openInWindow`?

Or do we want that in general surfaces can be transformed into forms?